moda-contracts/utils/NamespaceStorage.sol

```diff
@@ -0,0 +1,13 @@
+// SPDX-License-Identifier: MIT
+pragma solidity 0.8.21;
+
+contract NamespaceStorage {
```

Comment:
Is this used anywhere?

Reply:
This is just a helper I created that we can use to compute the storage location when using namespace storage in upgradeable contracts. This formula is defined in EIP-7201 https://eips.ethereum.org/EIPS/eip-7201

Reply:
Ok, let's only add code that we are currently using

Reply:
Ok removed.